### DISCUSSION TITLE
Values of Metrics in Absolute Nos or Percentage

### DISCUSSION THREAD

**Discussion by biased-ai** [pycaret/pycaret#3354] [Q&A]
Hi,

The .pull() method gives the model wise comparison of various metrics. My question is value like MAPE & SMAPE. Are these absolute in nature or by default expressed as percentage.
For eg, if the value of MAPE is 0.43, is it to assumed as 43% or just plain 0.43. So accuracy would be (100-0.43)

**Comment by ngupta23** [comment]
0.43 = 43%. Under the hood, it ends up calling sktime. You can check out the examples here:

https://github.com/sktime/sktime/blob/524597cee621a6dba35a5792acefdc0d3393ecfc/sktime/performance_metrics/forecasting/_functions.py#L1516-L1542
